tests/monitoring/component_monitoring.go

```diff
@@ -225,11 +218,10 @@ var _ = Describe("[sig-monitoring]Component Monitoring", Serial, decorators.SigM
 				_ = virtClient.VirtualMachineInstance(testsuite.NamespaceTestDefault).Delete(context.Background(), vmi.Name, metav1.DeleteOptions{})
 
 				g.Expect(libmonitoring.CheckAlertExists(virtClient, virtController.restErrorsBurtsAlert)).To(BeTrue())
```

Comment:
does this succeed locally?

Reply:
yes.. not sure why it fail again, will increase back the timeout maybe

Reply:
@machadovilaca ok now it failing also locally, any ideas?